rosidl_typesupport_opendds_c/resource/msg__type_support_c.cpp.em

```diff
@@ -199,7 +196,7 @@ if isinstance(type_, AbstractNestedType):
     size_t size = @(member.type.size);
 @[    else]@
     size_t size = ros_message->@(member.name).size;
-    if (size > (std::numeric_limits<DDS_Long>::max)()) {
+    if (size > (std::numeric_limits<int>::max)()) {
```

Comment:
Shouldn't this be DDS::Long or CORBA::Long, int will probably cause conversion warnings